content/docs/instrumenting/exporters.md

```diff
@@ -53,6 +53,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter)
    * [RabbitMQ Management Plugin exporter](https://github.com/deadtrickster/prometheus_rabbitmq_exporter)
    * [Mirth Connect exporter](https://github.com/vynca/mirth_exporter)
+   * [Kafka consumer group exporter](https://github.com/kawamuray/prometheus-kafka-consumer-group-exporter)
```

Comment:
Keep it sorted.

It'd also be good in your readme to point to the jmx exporter for all the other kafka stats